index.html

```diff
@@ -27,7 +27,20 @@
                 <a href="#"><img src="images/logomobile.svg" alt="Logo Mobile"></a>
             </div>
             <div class="menumobile">
-                <a href=""><img src="images/ic_menu.svg" alt="Mobile Menu"></a>
+                <nav class="navbar-mobile">
+                    <div class="hamburguer-menu">
+                        <div class="line line-1"></div>
+                        <div class="line line-2"></div>
+                        <div class="line line-3"></div>
+                    </div>
```

Comment:
- [ ] I cannot see the hamburger on the mobile version. I tried different browsers too. Please troubleshoot this issue.  
<img width="380" alt="Capture" src="https://user-images.githubusercontent.com/58504655/128016968-e4de593a-2200-471a-a7d8-427b8b8527b6.PNG">

Reply:
You have to use the "mobile" version of your navigator. 

![image](https://user-images.githubusercontent.com/84752859/128018141-45711a5a-f9fc-4f91-b4a8-b9fe31a2efba.png)